modules/packages/RangeChunk.chpl

```diff
@@ -84,6 +84,43 @@ module RangeChunk {
       else start..end;
   }
 
+  /*
+    Iterates through a range ``r``, which is blocked up in repeating ```numInds```
+    blocks of size ```blockSize```. Blocks are indexed from 0..numInds-1. 
+    Emits block at index ```idx``` in a cyclic manner.
```

Comment:
I found the names `idx` and `numInds` confusing for what they do. Maybe rename them to `tid` and `nTasks` or something more along those lines? More description in the header comment would also help.